libtextsecure/test/account_manager_test.js

```diff
@@ -40,6 +40,13 @@ describe('AccountManager', () => {
       });
     });
 
+    describe('do not encrypt null device name', () => {
```

Comment:
Given that the next item in the file is an `it()` why did you decide to go with a `describe()` here?

Reply:
It matched the pattern you had added above for encrypted device name.  Happy to change it as well.

Reply:
Okay, let's put your `it()` inside the `describe()` call right above where you were working, and call your test 'handles null deviceName'. Conceptually that `describe()` is about testing the `encryptDeviceName()` function.

Reply:
Done.